### DISCUSSION TITLE
Office 365 worked in 10.5.0, but not 10.5.1 or 10.5.2

### DISCUSSION THREAD

**Discussion by sthoen1024** [utmstack/UTMStack#730] [Q&A] [answered]
I discovered this pretty cool tool and had 10.5.0 working from the ISO install.  I setup O365 with little effort.  I had issues with the Meraki syslog not working, so when I saw 10.5.1 I decided to upgrade.  The O365 stopped working, but Meraki worked.  I figured it was so easy to setup, I deleted the UTMStack VM and re-installed 10.5.0 and immediately updated to 10.5.1.  Setup everything, again no O365.  So when 10.5.2 was released, I performed another VM from scratch, had everything working again, but not O365.  I decided to setup Wazuh and only setup O365 which worked without issue  - so I know the Tenant, Client ID and secret are all good.  How do I troubleshoot find logs to fix this?

**Comment by c3s4rfred** [comment]
Hi, @sthoen1024, try first to have at hand the docs at -> https://docs.utmstack.com/, this will let you know the system requirements and other important informations. Then check inside your instance, the Integrations menu and follow the guide of your desired integration. To see logs you can check the container logs using 'docker ps' and 'docker logs containerId' commands. In case of o365, it has a container for itself, so you can find logs there.

Best regards

**Comment by c3s4rfred** [comment, marked as answer]
Hi, @sthoen1024 a new release with fixes for o365 is out, please, update your instance to 10.5.3 and let us know if solve your issue.

Best regards

**Reply by sthoen1024** [reply]
Once the update was applied the O365 logs started to flow.
